src/data/constData.js

```diff
@@ -0,0 +1,132 @@
+export const CART_STATUS = {
+    itemsCount: 0,
+    itemsMRP: 0,
+    itemsAmount: 0
+}
+
+export const LOCATION = [
```

Comment:
LOCATION <> LOCATIONS
This is an array containing more than one values so please name accordingly